blocks/marketo-form/marketo-form.js

```diff
@@ -0,0 +1,143 @@
+const loadScript = (url, attrs) => {
+  const head = document.querySelector('head');
+  const script = document.createElement('script');
+  script.src = url;
+  if (attrs) {
+    // eslint-disable-next-line no-restricted-syntax, guard-for-in
+    for (const attr in attrs) {
+      script.setAttribute(attr, attrs[attr]);
+    }
+  }
+  head.append(script);
+  return script;
+};
+
+const formatFormLabel = (label) => {
+  const removeLabelAsterix = label.textContent.includes('*') ? label.textContent.slice(1) : label.textContent;
+  const removeLabelColon = removeLabelAsterix.includes(':')
+    ? removeLabelAsterix.substring(0, removeLabelAsterix.length - 1)
+    : removeLabelAsterix;
+
+  return removeLabelColon;
+};
+
+const embedMarketoForm = (marketoId, formId, successUrl, isHideLabels, block, formElement) => {
+  if (marketoId && formId) {
+    const mktoScriptTag = loadScript('//lp.pricefx.com/js/forms2/js/forms2.min.js');
```

Comment:
Can we not use the existing global `loadScript()` method to load the external js?.

Reply:
And also `forms2.min.js` is loading multiple times. It should load only once. To avoid that you can use the global method to load the js only once in a page.
<img width="1248" alt="image" src="https://github.com/pricefx/pricefx-eds/assets/161038987/8d77d1ce-2c6b-43e9-a9f4-68ebd71037ae">